Makefile

```diff
@@ -26,7 +26,7 @@ makemigrations:
 	docker-compose run web python manage.py makemigrations
 
 pyupgrade:
-	docker run --rm -v /$$(pwd):/data quay.io/watchdogpolska/pyupgrade
+	docker run --rm -v /$$(pwd)/backend-project:/data quay.io/watchdogpolska/pyupgrade
```

Comment:
That would work properly on GitHub, but for developers convenience, I added `/backend-project` here as `node_modules/` installed for Ant Design cause something like that:
```
magul@bohr small_eod (fixing-github-actions *%) $ make pyupgrade 
docker run --rm -v /$(pwd):/data quay.io/watchdogpolska/pyupgrade
Rewriting ./frontend-project/design_pro/node_modules/railroad-diagrams/railroad_diagrams.py
Rewriting ./frontend-project/design_pro/node_modules/node-pty/deps/winpty/ship/make_msvc_package.py
Rewriting ./frontend-project/design_pro/node_modules/node-pty/deps/winpty/misc/DebugClient.py
make: *** [Makefile:29: pyupgrade] Error 123
```